src/devServer.ts

```diff
@@ -31,7 +31,7 @@ export function startDevServer(config: Config = {}) {
     res.statusCode = 500;
     res.end();
   });
-  const port = config.devServer?.port ?? 3000;
+  const port = config.devServer?.port ?? process.env.PORT ?? 3000;
```

Comment:
I have to add `PORT` here, otherwise it's pretty hard to modify the port with multiple server instances

Reply:
Okay, it's a little unfortunate, but I will revisit it later anyway.